app/templates/components/public/sponsor-list.hbs

```diff
@@ -1,4 +1,4 @@
-<h3 class="ui header">{{t 'Sponsors'}}</h3>
+<h3 class="ui header"/>
```

Comment:
why keeping this line, remove it.

Reply:
if I remove this line, then any other components using this component will be affected. This is what I understood from the previous comment by @iamareebjamal.
If that's incorrect then I can delete the whole <h3/> line and make a pull request.